contracts/contracts/oracle/AeroWethPriceFeed.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/chainlink/AggregatorV3Interface.sol";
+import { StableMath } from "../utils/StableMath.sol";
+
+// @notice Oracle to fetch AERO Token price that denominates in WETH
+contract AeroWEthPriceFeed {
+    using StableMath for uint256;
+
+    AggregatorV3Interface public immutable wethUsdPriceFeed;
+    AggregatorV3Interface public immutable aeroUsdPriceFeed;
+
+    address public constant AERO_TOKEN_ADDRESS =
+        0x940181a94A35A4569E4529A3CDfB74e38FD98631;
+    address public constant WETH_TOKEN_ADDRESS =
+        0x4200000000000000000000000000000000000006;
+
+    constructor(
+        AggregatorV3Interface _wethUsdPriceFeed,
+        AggregatorV3Interface _aeroUsdPriceFeed
+    ) {
+        wethUsdPriceFeed = _wethUsdPriceFeed;
+        aeroUsdPriceFeed = _aeroUsdPriceFeed;
+    }
+
+    /**
+     * @notice Returns the total price in 18 digit units for the given asset.
+     *         This implementation does not (!) do range checks as the
+     *         parent OracleRouter does.
+     * @param asset address of the asset
+     * @return uint256 unit price for 1 asset unit, in 18 decimal fixed
+     */
+    function price(address asset) external view returns (uint256) {
```

Comment:
A price feed usually communicates a price rate between 2 different assets. There is no notion of a base asset (e.g. ETH or USD). Similar to how it is done [here](https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/oracle/AuraWETHPriceFeed.sol#L45).

For that reason the price shouldn't be accepting the asset parameter